templates/deployment.yaml

```diff
@@ -40,7 +40,10 @@ objects:
         },
         "branding": {
           "appName": "Hawtio Console",
-          "appLogoUrl": "img/hawtio-logo.svg"
+          "appLogoUrl": "img/hawtio-logo.svg",
+          "linkText": "Hawtio Console",
+          "linkSection": "Hawtio",
+          "linkImagePath": "/online/img/favicon.ico"
```

Comment:
I assumed the image would always be included in the web app, therefore, all we needed would be the path to that image. Do you want to support full URLs as well?

Reply:
In case we decide the image would always be included in the web app, then I think `imageRelativePath` would make sure it's relative to the web app route hostname. If we want let users use their own image, we may go for supporting arbitrary absolute URLs.